docs/_articles/en/faq.md

```diff
@@ -21,16 +21,21 @@ A: No! `Salesforce CLI Scanner` can be used on any codebase.
 ## Questions about language support
 
 #### Q: What languages does `Salesforce CLI Scanner` support?
-A: By default, `Salesforce CLI Scanner` supports code written in Apex, VisualForce, Java, JavaScript, and TypeScript. You can extend it to support any language.
+A: By default, `Salesforce CLI Scanner` supports code written in Apex, VisualForce, Java, JavaScript, XML, PL/SQL and TypeScript.
 
-#### Q: How do I add a language to `Salesforce CLI Scanner`?
-A: The file types targeted by each rule engine are defined in `~/.sfdx-scanner/Config.json` in the `targetPatterns` property for each entry.
+#### Q: How do get support to additional languages?
+Please create an Issue on our [Github repo](https://github.com/forcedotcom/sfdx-scanner). We'll try to help you out. 
 
-To make a particular rule engine scan a new language, add that language's file extension to the `targetPatterns` property for that rule engine in `~/.sfdx-scanner/Config.json`. For example, to start scanning Python files with PMD, add `**/*.py` to the `targetPatterns` property for PMD.
+#### Q: How do I add a supported language to `Salesforce CLI Scanner`?
+A: By default, supported languages are added. If you still need, follow the steps below.
+
+The file types targeted by each rule engine are defined in `~/.sfdx-scanner/Config.json` in the `targetPatterns` property for each entry.
+
+To make a particular rule engine scan a new language (from the set of [supported languages](./en/troubleshooting/#supported-languages-for-pmd)), add that language's file extension to the `targetPatterns` property for that rule engine in `~/.sfdx-scanner/Config.json`. For example, to start scanning Python files with PMD, add `**/*.py` to the `targetPatterns` property for PMD.
```

Comment:
I think we should remove this section, since we pruned the jars, it won't work.

Reply:
ok, makes sense.

Reply:
The only reason I kept it was if they removed any language and want to add it back.

Reply:
I removed the question on how to remove a language too.